gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -33,8 +33,8 @@ export default function NxGlobalSidebarPage() {
           <NxList.Item>
             <NxList.Text>
               A footer which contains meta-information about the application. Refer to the
-              {' '}<a className="nx-text-link" href="#/Pages/NxGlobalSidebarFooter">NxGlobalSidebarFooter</a> page for
-              its documentation.
+              {' '}<a className="nx-text-link" href="#/pages/Global%20Sidebar%20Footer">NxGlobalSidebarFooter</a> page
+              {' '}for its documentation.
```

Comment:
`{' '}` is not necessary when the line break is between two hardcoded strings
```suggestion
              for its documentation.
```

Reply:
> `{' '}` is not necessary when the line break is between two hardcoded strings

fixed https://github.com/sonatype/sonatype-react-shared-components/pull/732/commits/2af909c1b453a2c1513bf077d4f56aa9b2817ffd